### PR TITLE
chore(frontend): make shuffle box full width

### PR DESCRIPTION
**Pull request by edalex-yinzi**
##### Checklist

- [x] the [contributor license agreement][] is signed
- [x] commit message follows [commit guidelines][]
- [ ] tests are included
- [x] screenshots are included showing significant UI changes
- [ ] documentation is changed or added

##### Description of change
Make shuffle box full width.

image

[contributor license agreement]: https://www.apereo.org/node/676
[commit guidelines]: https://chris.beams.io/posts/git-commit/

### CODE REVIEW

File: react-front-end/tsrc/components/ShuffleBox.tsx [hunk 1]
```diff
@@ -161,7 +161,7 @@ export const ShuffleBox = ({
 
   return (
     <Grid id={id} container spacing={2}>
-      <Grid item xs={3}>
+      <Grid item xs>
```
Comment (edalex-ian) [inline react-front-end/tsrc/components/ShuffleBox.tsx:164]: Do you need `xs` at all in this case? Can it just be:

```suggestion
      <Grid item>
```

(Same query for other one below too.)

**Review by PenghaiZhang — APPROVED** [review]
LGTM. Just remove the two `xs` if they are not used.

File: react-front-end/tsrc/components/ShuffleBox.tsx (hunk 1 above)
Comment (edalex-yinzi) [reply react-front-end/tsrc/components/ShuffleBox.tsx:164]: I had the same idea before but unfortunately, we need this. :smile: 
https://mui.com/components/grid/#auto-layout

**Comment by edalex-yinzi** [comment]
We do need xs to keep the right and left boxes in the same width.

**Pull request closed without merging**